astropy/table/tests/test_table.py

```diff
@@ -909,6 +909,55 @@ def test_remove_row_preserves_meta(self, table_types):
         assert self.t.dtype == np.dtype([(str('a'), 'int'),
                                          (str('b'), 'int')])
 
+    
+    def test_remove_masked_rows_1(self, table_types):
+        self._setup(table_types)
+        self.t.add_column(self.b)
+        self.t.add_column(self.c)
+        self.t.remove_masked_rows()
+        assert self.t.colnames == ['a', 'b', 'c']
+        assert np.all([self.t['a'] == np.array([1, 2, 3])])
+        assert np.all([self.t['b'] == np.array([4, 5, 6])])
+        assert np.all([self.t['c'] == np.array([7, 8, 9])])
+
+    def test_remove_masked_rows_2(self):
+        z = """
+        a,b,c
+        ,2.0,3.11
+        1.34,2.12,3.1
+        89.12,33.24,11.22
+        12,1.17,
+        """
+        
+        t = ascii.read(z)
```

Comment:
I would suggest to use the format for reading that is used elsewhere in this file, it makes it a tiny bit more readable e.g.: https://github.com/astropy/astropy/blob/master/astropy/table/tests/test_table.py#L1312

Reply:
so just make it `t = table.Table.read(z)` ?

Reply:
Could you tell me what to do about the changelog entry? Do I have to edit something? This feature doesn't seem like too big of a change in the code base.